driver-core/src/main/java/com/datastax/driver/core/querybuilder/QueryBuilder.java

```diff
@@ -351,6 +351,19 @@ public static Clause in(String name, Object... values) {
      * @param values the values
      * @return the corresponding where clause.
      */
+    public static Clause in(String name, Collection<?> values) {
+        return new Clause.InClause(name, values);
+    }
+
+    /**
```

Comment:
I considered removing this method, but per the binary compatibility guide [13.4.14](https://docs.oracle.com/javase/specs/jls/se7/html/jls-13.html#jls-13.4.14):

> Changing [...] the type of a formal parameter [...] creates a method or constructor with a new signature, and has the combined effect of deleting the method or constructor with the old signature and adding a method or constructor with the new signature (§13.4.12).

Clirr throws an error as well, so figured would just deprecate it instead.